tools/Vrops.py

```diff
@@ -64,10 +64,10 @@ def get_adapter(self, target: str, token: str, adapterkind: str) -> (list, int):
                                     timeout=30)
         except requests.exceptions.ReadTimeout as e:
             logger.error(f'Request to {url} timed out. Error: {e}')
-            return adapter, 504
+            return adapter, 504, 0
```

Comment:
Why don't we use the real time here? Or we return the timeout value, since it should be identical.

Reply:
At this point, there is no response object. The request aborts early.

Reply:
Is it exited with the 30seconds timeout or somewhen in between?